src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,17 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+
+export const TodoInfo = ({ todo }) => {
+  const { title, completed, user } = todo;
+
+  return (
+    <article className={cn('TodoInfo', {
+      'TodoInfo--completed': completed === true,
```

Comment:
you don't need to compare if completed is true, because it already gives the result of true or false.

Reply:
Good point 👍 
Please fix it